js/src/common/constants/index.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export const ALTERNATE_LOCATION = 'inspector';
+export const DEFAULT_LOCATION = 'editor';
+export const LOCATIONS = [ DEFAULT_LOCATION, 'inspector' ];
```

Comment:
I'm not sure I like this. 🤔 

What's the purpose of abstracting "locations"? Using "Alternate" implies there could only be two locations, which is true… so why not just call them Editor and Inspector?

Reply:
There are a few places where it's good to know that it's a default location:

https://github.com/studiopress/genesis-custom-blocks/blob/b1062267f4166726ce80581ab26fff5d0aaaf08e/js/src/edit-block/hooks/useField.js#L204-L217

https://github.com/studiopress/genesis-custom-blocks/blob/b1062267f4166726ce80581ab26fff5d0aaaf08e/js/src/edit-block/components/editor.js#L84

But I could change it if you'd like. As long as the value of those constants stays the same, it'll work the same whether it's named `EDITOR_LOCATION` or `DEFAULT_LOCATION`

Reply:
Of course. Totally your call on this one. Feel free to resolve.

Reply:
If it's alright, https://github.com/studiopress/genesis-custom-blocks/pull/64/commits/60337ef885d98e5fe4a121295db889a3f79c165b renames `ALTERNATE_LOCATION` to `INSPECTOR_LOCATION`, and replaces 2 uses of `DEFAULT_LOCATION` with `EDITOR_LOCATION`. 

Good call on that, default and alternate don't really describe them.